.gitignore

```diff
@@ -5,3 +5,4 @@
 /doc/
 /pkg/
 /tmp/
+.idea
```

Comment:
@TheBlackArroVV thoughts about adding `.idea` into your global gitignore instead of gem's one?

Cause it's really needed just for you, not for the project

Reply:
Also, next time a project doesn't have it in the `.gitignore` there is a risk to commit this folder.